src/styles/TopNavStyles.ts

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import AppBar from '@material-ui/core/AppBar';
+import IconButton from '@material-ui/core/IconButton';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+
+export const Root = styled.div`
+    flex-grow: 1;
+`;
+
+export const Bar = styled(AppBar)`
+    position: sticky;
+    background: transparent;
+    box-shadow: none;
+`;
+
+export const ToolBar = styled(Toolbar)`
+    && { min-height: 128px;
+    align-items: flex-start;
+    padding-top: 8px;
+    padding-bottom: 16px;
+    background: white;
+    }
+`;
+
+export const MenuButton = styled(IconButton)`
```

Comment:
It doesn't look like the styles for MenuButton, AccountButton, or ListStyle are reflected/applied (probably lower level of specificity than the MUI built-in styling). 

These components also respond to an MUI theme for coloring/other styling, which Tiffany has started on her PR -- don't worry about these components for now. You can remove these three and just use the raw MUI components